src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -101,10 +101,12 @@ const props = defineProps<Props>();
 
 const state = reactive({
     hasManagePermission: useManagePermissionState(),
-    dashboardId: computed<string>(() => props.dashboardId),
-    dashboardViewer: DASHBOARD_VIEWER.PRIVATE as DashboardViewer,
-    dashboardName: 'Dashboard',
-    labelList: [] as Array<string>,
+    dashboardConfig: DASHBOARD_TEMPLATES.monthlyCostSummary, // TODO: should be changed to api data
```

Comment:
nitpick: I think `dashboardInfo` is more suitable. `DashboardConfig` has no `viewers`, but `DashboardModel` has.